ui/webui/src/helpers/utils.js

```diff
@@ -38,3 +40,19 @@ export const checkIfArraysAreEqual = (array1, array2) => {
         array1Sorted.every((value, index) => value === array2Sorted[index])
     );
 };
+
+/* Converts an object with variant values to an object with normal values
+ * @param {Object} dbusObject
+ * @returns {Object} The converted object
+ */
+export const objectFromDBus = (dbusObject) => {
+    return Object.keys(dbusObject).reduce((acc, k) => { acc[k] = dbusObject[k].v; return acc }, {});
+};
+
+/* Converts an object with normal values to an object with variant string values
+ * @param {Object} object
+ * @returns {Object} The converted object
+ */
+export const objectToDBus = (object) => {
```

Comment:
Maybe we should indicate in the name that the function produces string variants.